src/julia-parser.scm

```diff
@@ -102,12 +102,12 @@
                                       0))
 
 (define unary-ops (append! '(|<:| |>:|)
-                           (add-dots '(+ - ! ~ ¬ √ ∛ ∜))))
+                           (add-dots '(+ - ! ~ ¬ √ ∛ ∜ ⋆ |⋆.|))))
```

Comment:
You don't need `|⋆.|` here because `add-dots` adds it.

Reply:
That is incorrect, it is needed here, because without it, the dot syntax doesn't work
```Julia
julia> ⋆(x) = x
⋆ (generic function with 1 method)

julia> ⋆.(1,2,3)
ERROR: syntax: invalid identifier name "."

julia> ⋆.(x...) = broadcast(⋆,x)
ERROR: syntax: invalid identifier name "."
```
therefore, I am going to leave it in there and will not change it

Reply:
That's because you put the dot in the wrong place.  `⋆.` should not be an operator, only `.⋆`, just like all the rest of our operators.

Reply:
Ah, okay, I got a bit confused there